tests/integration/linux_bridge_test.py

```diff
@@ -537,6 +539,37 @@ def test_disable_vlan_filtering_on_bridge(
             assertlib.assert_state_match(desired_state)
 
 
+def test_bridge_with_unmanaged_ports():
+    bridge_config = _create_bridge_subtree_config([])
+    with linux_bridge(TEST_BRIDGE0, bridge_config):
+        with dummy0_as_slave(TEST_BRIDGE0):
```

Comment:
The test itself is about detecting unmanaged slaves when reporting the state.
Therefore, the test setup belongs to a fixture IMO.

Reply:
Done, thanks!